CMakeLists.txt

```diff
@@ -87,7 +88,7 @@ add_subdirectory(src)
 
 
 ## Executable targets
-find_package(Boost 1.51 REQUIRED COMPONENTS program_options)
+find_package(Boost 1.51 REQUIRED COMPONENTS program_options thread)
```

Comment:
`Boost::thread` is not used

Reply:
but `boost::spirit` uses it in 1.58. if you dont link it you get all kinds of linker errors when you do the `-DBOOST_SPIRIT_THREADSAFE`. what am i misunderstanding here?

Reply:
`boost::spirit` requires `boost::thread` but it is not right place to to add the requirement. 
The `boost::thread` must a public link dependency of the `valhalla` target.